cmd/notary-server/config.json

```diff
@@ -3,10 +3,7 @@
 		"addr": ":4443"
 	},
 	"trust_service": {
-		"type": "remote",
```

Comment:
We should leave config.json as is. I think Richard is using if for deployments. dev-config.json should represent an in memory setup for notary-server.